tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -252,6 +252,14 @@ export class ScalarCardComponent<Downloader> {
     );
   }
 
+  canExpandTable() {
+    const visbleRuns = this.dataSeries.filter((datum) => {
+      return isDatumVisible(datum, this.chartMetadataMap);
+    });
+
+    return visbleRuns.length > 3;
```

Comment:
Can you add a comment explaining how 3 runs fill the _data_table_initial_height.